pkg/klock/klock.go

```diff
@@ -66,6 +67,19 @@ func (o Options) Validate() error {
 	}
 }
 
+func (o Options) NormalizedLabelColumns() []string {
+	labelCols := []string{}
+	for _, labels := range o.LabelColumns {
+		for _, label := range strings.Split(labels, ",") {
+			label = strings.TrimSpace(label)
+			if len(label) > 0 {
+				labelCols = append(labelCols, label)
+			}
```

Comment:
The `TrimSpace` and skipping empty entries is nice, but I'd rather have 1:1 behaviour with `kubectl` in this case. Suggest just using the values as-is, and skipping the normalization completely.